clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java

```diff
@@ -213,7 +214,7 @@ public FileOutStream getOutStream(long fileId, ClientOptions options) throws IOE
    */
   public boolean mkdirs(TachyonURI path) throws FileAlreadyExistException, InvalidPathException,
       IOException {
-    return mkdirs(path, true);
+    return mkdirs(path, false);
```

Comment:
Sounds good. Add the following TODO for the mkdirs function:

``TODO(calvin,jiri): Consider renaming to mkdir``